tests/Resources/ti.platform.test.js

```diff
@@ -225,6 +231,13 @@ describe('Titanium.Platform', function () {
 
 	it('.macaddress', () => {
 		should(Ti.Platform).have.readOnlyProperty('macaddress').which.is.a.String();
+		if (OS_IOS) {
+			const platformId = Ti.Platform.id;
+			should(platformId).be.a.String();
+			should(platformId.length).eql(36);
+			// Verify format using regexp!
+			should(platformId.match(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i)).not.eql(null);
```

Comment:
The test here is for the `macaddress` property, but you're checking the `Ti.Platform.id` property.

Reply:
Copy paste issue :). Fixed.